artifacts_docker_test.go

```diff
@@ -0,0 +1,11 @@
+// +build docker
+
+package main
+
+import "github.com/taskcluster/slugid-go/slugid"
+
+var (
+	// all tests can share taskGroupId so we can view all test tasks in same
+	// graph later for troubleshooting
+	taskGroupID = slugid.Nice()
```

Comment:
This is already a feature of the generic-worker tests, why is this logic duplicated?

Reply:
`artifacts_test.go` in docker builds

Reply:
I fixed it by moving taskGroupID to its own module.